File: Task3/task3.py
n = int(input("Введите количество чисел N: "))
print(f"Введите {n} чисел через пробел (или по одному на строку):")

numbers = []
while len(numbers) < n:
    numbers.extend(input().split())

count = sum(1 for num in numbers[:n] if len(num) % 2 == 0)
print("Результат:", count)

File: Task3/task3.swift
import Foundation

print("Введите количество чисел N:")
guard let nStr = readLine(), let n = Int(nStr) else {
    fatalError("Неверный ввод")
}

print("Введите \(n) чисел через пробел:")
guard let numbersLine = readLine() else {
    fatalError("Нет данных")
}

let numbers = numbersLine.split(separator: " ")
var count = 0
for i in 0..<n {
    let num = i < numbers.count ? String(numbers[i]) : ""
    if num.count % 2 == 0 {
        count += 1
    }
}

print("Результат: \(count)")